### PR TITLE
EPMDJ-2427: Implement the functionality of table sorting

### PR DESCRIPTION
**Pull request by RomanAchilov**
Issue: EPMDJ-2427

### CODE REVIEW

File: src/hooks/use-sorter.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react';
+
+export function useSorter(data: any[], columnName: string, order: 'ASC' | 'DESC'): any[] {
+  const [sortedData, setSortedData] = useState<any[]>(data);
+
+  const getSortType = () => {
```
Comment (pavyarov) [inline src/hooks/use-sorter.ts]: replace to reducer or pass descriptor as prop

File: src/pages/agent-full-page/coverage-plugin/coverage-details/coverage-details.tsx [hunk 2]
```diff
@@ -78,14 +86,57 @@ export const CoverageDetails = coverageDetails(
                 name="name"
                 label="Name"
                 Cell={({ value }) => <NameCell icon={<Icons.Package />} value={value} />}
+                HeaderCell={() => (
+                  <SortArrows
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/coverage-details/coverage-details.tsx]: move to table

File: src/pages/agent-full-page/coverage-plugin/coverage-details/coverage-details.tsx [hunk 3]
```diff
@@ -19,20 +22,25 @@ interface Props {
...
 const coverageDetails = BEM(styles);
 
 export const CoverageDetails = coverageDetails(
-  ({
-    className, coverageByPackages, header, associatedTestsTopic,
+  ({
+    className, coverageByPackages, header, associatedTestsTopic,
   }: Props) => {
     const [selectedId, setSelectedId] = React.useState('');
+    const [order, setOrder] = React.useState<Order>('ASC');
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/coverage-details/coverage-details.tsx]: should be stored at table

**Comment by RomanAchilov** [comment]
Updated useSorter hook in storybook

**Pull request closed without merging**